components/speedreader/rust/lib/examples/feature_extraction_streamer.rs

```diff
@@ -19,6 +19,11 @@ fn main() {
     streamer.write(&mut frag3.as_bytes()).unwrap();
 
     let sink = streamer.end();
+    sink.rcdom.on_parse_error = Some(Box::new(|errors| {
+        // parsing errors may happen due to malformed HTML, but it will not affect
+        // parsing itself or the DOM tree building
+        println!("Errors: {:?}", errors);
```

Comment:
Probably we don't want general print statements in Brave Core.  Is there a LOGV or equiv for rust?

Reply:
this is inside the example module. core doesn't call this code. just keeping parity with the old code since i ported the DOM to a newer (safer) library.

Reply:
The important files for the Arc90 stuff are `src/readability/extractor.rs`, `src/readability/scorer.rs`, and `src/readability/dom.rs` (utils for the scorer).